digitizer/config.py

```diff
@@ -46,8 +46,9 @@
 def find_by_name(name, json):
     """Find JSON corresponding to quantity name."""
     for q_json in json:
-        if q_json['name'] == name:
+        if q_json['name'] == name or name in q_json["synonyms"]:
             return q_json
+    # TODO: if find_by_name is used for json structure without "synonyms" attribute, update if/then
```

Comment:
Ok, this would be quite easy to do (note: just typed down, not tested)

```python
try:
  names = [ q_json['name'] ] + q_json['synonyms']
except AttributeError:
  names = [ q_json['name'] ]

if name in names:
  return q_json
```

Reply:
Let me test this. I cannot remember if I came up with a solution that hasn't been added to the PR.

Reply:
This works as expected, so I committed the changes and the PR is updated.